### DISCUSSION TITLE
Icon always on top in trackNode rendering order...

### DISCUSSION THREAD

**Discussion by keince** [pelicanmapping/osgearth#1897] [Support Q&A] [answered]
I have a trackNode with an icon and several added drawables within the trackNode to render different types of symbology along with the icon. 

Is there a way to make it so the trackNode icon is always on top of its drawables? Right now, the drawables render on top of the icon.

Any help would be appreciated...

**Comment by gwaldron** [comment, marked as answer]
Doesn't look like it. Someone would need to add this as a feature. I will convert this to an Issue ticket.

**Reply by keince** [reply]
Okay. Thanks for the feedback and submitting the ticket. 

I guess I will need to come up with something else in the meantime...
